challenge_dishwasher/src/challenge_dishwasher/custom_place.py

```diff
@@ -57,7 +57,7 @@ def place_pose(ud):
 
         @cb_interface(outcomes=['done'])
         def open_gripper(ud):
-            arm.resolve().send_gripper_goal("open", timeout=0)
+            arm.resolve().gripper.send_goal("open", timeout=0)
```

Comment:
Right now I keep the publicarm interface backwards compatible with the old interface. I added deprecation warnings to notify the user of the changing interface. I have changed a bunch of smach states and challenges already to use the new interface.  so that the deprecation warnings wont drive us crazy.